server.js

```diff
@@ -315,7 +321,7 @@ function how_many_games_to_queue(max_games, w_obs, l_obs, pessimistic_rate) {
 
 app.enable('trust proxy');
 
-app.use(bodyParser.urlencoded({extended: true}));
+//app.use(bodyParser.urlencoded({extended: true}));
```

Comment:
Is bodyParser not actually needed? I added it on long ago when first starting this project, just based on some examples I as starting out from. (This was my first Node project from scratch and first use of Express.) We don't lose access to req.body in various locations with it gone?